Source/Core/DolphinQt/ConvertDialog.cpp

```diff
@@ -257,6 +243,7 @@ void ConvertDialog::OnFormatChanged()
   m_block_size->setEnabled(m_block_size->count() > 1);
   m_compression->setEnabled(m_compression->count() > 1);
 
+  // Block scrubbing of Datel discs when using non-RVZ container
```

Comment:
This comment is misleading. We block scrubbing when we have a Datel disc **or** when we have an RVZ container.